pages/desktop/base.py

```diff
@@ -242,7 +242,7 @@ class Footer(Region):
     _root_locator = (By.CSS_SELECTOR, '.Footer-wrapper')
     _footer_amo_links_locator = (By.CSS_SELECTOR, '.Footer-amo-links')
     _footer_browsers_links_locator = (By.CSS_SELECTOR, '.Footer-browsers-links')
-    _footer_products_links_locator = (By.CSS_SELECTOR, '.Footer-product-links')
+    _footer_products_links_locator = (By.CSS_SELECTOR, '.Footer-wrapper section:nth-child(4)')
```

Comment:
I found a locator that works both for AMO and DevHub. How I can check the Product footer links from the same tests for both pages.